internal/metarepos/report_collector_test.go

```diff
@@ -602,6 +603,76 @@ func TestReportDedup(t *testing.T) {
 	})
 }
 
+func TestReportHandleInvalidReport(t *testing.T) {
+	a := NewDummyStorageNodeClientFactory(1, true)
+	mr := NewDummyMetadataRepository(a)
+
+	logger, _ := zap.NewDevelopment()
```

Comment:
Please use the `-v` test flag with zaptest to control verbosity.

Reply:
`zaptest` looks good. thanx